packages/docprovider/src/yprovider.ts

```diff
@@ -37,8 +42,9 @@ export class WebSocketProvider implements IDocumentProvider {
     this._contentType = options.contentType;
     this._format = options.format;
     this._serverUrl = options.url;
-    this._ydoc = options.model.ydoc;
+    this._sharedModel = options.model;
     this._awareness = options.model.awareness;
+    this._trans = options.translator ?? nullTranslator.load('jupyterlab');
```

Comment:
Small fix:

```suggestion
    this._trans = (options.translator ?? nullTranslator).load('jupyterlab');
```

Reply:
Ouch. I committed this change. But it was OK.

`options.translator` is already a TranslationBundle. We do not need to call `options.translator.load('jupyterlab')`. This is done in the activation function of the plugin.

Reply:
oh yes sorry I forget about that.